kahuna/public/js/upload/view.html

```diff
@@ -22,27 +22,14 @@
 
     <file-prompt class="section" ng:if="!ctrl.latestJob"></file-prompt>
 
-    <section class="section">
+    <section class="section" ng:if="!ctrl.latestJob">
```

Comment:
Yeah. Its so that the recent uploads isn't displayed when we're currently uploading. It makes the UI a bit simpler/cleaner imo.